cmake/BuildTests.cmake

```diff
@@ -120,29 +123,30 @@ FOREACH(testfile ${UNIT_TESTS})
 ENDFOREACH()
 message("-- Finished building ${UNIT_TEST_COUNT} unit test file(s)...")
 
-if(NOT WIN32)
+if(NOT WIN32 AND ENABLE_NANOFI)
```

Comment:
Can you increase the indentation of the contents, until line 147?

Reply:
Added in 2847fc59b50d13188dd9aa9ddf1c1fd5aad4905f